cm06/scan.py

```
@@ -0,0 +1,243 @@
+#!/usr/bin/env python3
+import argparse
+import datetime
+import enum
+import json
+import logging
+import os
+
+import semver
+
+import ci.log
+import cnudie.iter
+import cnudie.retrieve
+import delivery.client
+import dso.labels
+import dso.model
+import ocm
+
+import ctx_util
+import lookups
+import rescore.model
+import rescore.utility
+
+
+logger = logging.getLogger(__name__)
+ci.log.configure_default_logging()
+
+own_dir = os.path.abspath(os.path.dirname(__file__))
+default_cache_dir = os.path.join(own_dir, '.cache')
+
+
+class AnalysisLabel(enum.StrEnum):
+    LINT = 'lint'
+    SAST = 'sast'
+
+
+def load_json_file(
+    file_path: str
+) -> dict:
+    try:
```

at any rate, I think we can shortcut this kind of fine-granular exception-handling:

- open will already raise an exception (file-not-found) -> there is little benefit from printing "file-not-found", and re-raising it (just let `open` do its job, and let it raise..)
- ditto for JSONDE